script/cibuild

```diff
@@ -27,8 +31,12 @@ exec_ci() {
 
 main() {
   import_dependencies
-  install_texlive
   install_knitr
+  if [[ "${TEX_DIST}" == "miktex" ]]; then
```

Comment:
If we change this to `"${TEX_DIST:=texlive}"` we can remove the explicit defaults in `.travis.yml` and `.appveyor.yml`.
Unless you object, I'll tweak that while debugging the build.